spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/tf/TensorflowToBigDL.scala

```diff
@@ -396,6 +396,71 @@ object Conv2D extends TensorflowToBigDL{
   }
 }
 
+object Conv2D2 extends TensorflowToBigDL{
+  private val graph = {
+    val add = Node("Add")
+    val conv = Node("Conv2D")
+    val reshape = Node("Reshape")
+
+    Node("*") -> conv
+    Node("Const") -> Node("Identity") -> conv -> add
+    Node("Const") -> Node("Identity") -> reshape
+    Node("Const") -> reshape
+    reshape -> add
+
+    add.graph(reverse = true)
+  }
+
+  override def topology: DirectedGraph[String] = graph
+
+  override def layer[T: ClassTag](tfGraph: DirectedGraph[NodeDef],
+    context: Context[T],
+    byteOrder: ByteOrder)(
+    implicit ev: TensorNumeric[T]): AbstractModule[Activity, Tensor[T], T] = {
+
+    val attributes = tfGraph.source.prevNodes(0).element.getAttrMap
+    require(attributes.get("strides").getList.getI(0).toInt == 1, s"not support strides on batch")
+    require(attributes.get("data_format").getS.toString(Charset.defaultCharset()) == "NCHW",
+      "NCHW should be used for this sub-graph")
+
+    require(attributes.get("strides").getList.getI(2).toInt == 1, s"not support strides on depth")
+    val (strideH, strideW) = (attributes.get("strides").getList.getI(2).toInt,
+      attributes.get("strides").getList.getI(3).toInt)
+
+    val biasNode = tfGraph.source.prevNodes(1).prevNodes(0).prevNodes.head.element
+    val (bias, gradBias) = getOrSetTensor(biasNode, context, byteOrder)(t => t)
```

Comment:
It seems the `Reshape` operation is not reflected on the BigDL model. Should we impose some requirement on the shape of this biasTensor?